django_filters/utils.py

```diff
@@ -177,7 +173,7 @@ def verbose_field_name(model, field_name):
     names = []
     for part in parts:
         if isinstance(part, ForeignObjectRel):
-            names.append(force_text(part.related_name))
+            names.append(part.related_name.replace('_', ' '))
```

Comment:
btw - I don't think `force_text` was necessary here, since it's just the attribute accessor name.